SmartVision/WindowsForms/FaceDetection.cs

```diff
@@ -42,5 +42,26 @@ public static void DisableFaceDetection()
             }
         }
 
+        /// <summary>
+        /// Detects a face in a picture and returns an answer
+        /// </summary>
+        public static bool FaceDetectionFromPicture(Bitmap faceImageBitmap)
+        {
+            Image<Bgr, byte> image = new Image<Bgr, byte>(faceImageBitmap);
+            var grayFrame = image.Convert<Gray, Byte>();
+            var faces = cascade.DetectMultiScale(grayFrame, 1.1, 10, Size.Empty);
+
+            if (faces.Length > 0)
```

Comment:
Maybe you should check if faces.Length == 1? Because if there were a few faces in a single picture, how we'll determine which one is from the missing person?